.github/workflows/preview-env-teardown.yml

```diff
@@ -64,10 +57,10 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
 
       - name: Update comment with tear-down status
-        if: steps.find-comment.outputs.comment-id != ''
-        uses: peter-evans/create-or-update-comment@v4
+        uses: mshick/add-pr-comment@v2
         with:
-          comment-id: ${{ steps.find-comment.outputs.comment-id }}
-          body: |
-            :broom: Preview environment for this PR has been torn down. <!-- preview-env -->
-          edit-mode: replace
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          message-id: preview-env-status
```

Comment:
`message-id` is also not necessary, if you don't plan on having more than one comment you want to update throughout the process. There's a default value like `add-pr-comment`.